scripts/print-report.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/local/bin/bash
+
+pretty_print_countries() {
+  local failed_tests=($(cat $1))
+  local failed_countries=($(for test in ${failed_tests[@]}; do [[ $test =~ ^.*\/(.*)\.feature:.*$ ]] && echo ${BASH_REMATCH[1]}; done))
+  local country_names=${failed_countries[@]^}
+  echo ${country_names// /, }
+}
+
+[[ -z "$1" ]] && echo "Environment not set" && exit 1
+
+ENVIRONMENT="$1"
+
+pn_countries=($(pretty_print_countries proxy-node-report-"$ENVIRONMENT"/failed-tests))
+cn_countries=($(pretty_print_countries connector-node-report-"$ENVIRONMENT"/failed-tests))
+
+num_failed_pn=${#pn_countries[@]}
+num_failed_cn=${#cn_countries[@]}
+[[ $num_failed_pn -eq 0 && $num_failed_cn -eq 0 ]] && exit 0
+
+echo "There are $num_failed_pn PN and $num_failed_cn CN broken connections with eIDAS countries in $ENVIRONMENT"
```

Comment:
Minor thing - this could be confusing if only the PN or the CN has failing tests. As you'd only see one line printed by the lines below.